conf-sample.js

```diff
@@ -7,6 +7,7 @@ c.mongo.port = 27017
 c.mongo.db = 'zenbot4'
 c.mongo.username = null
 c.mongo.password = null
+c.mongo.replicaSet = ''
```

Comment:
fair, I added a comment hopefully explaining it and the implementation of it enough.